### PR TITLE
Handle Translation & Multi-Language

### PR DESCRIPTION
**Pull request by IQbrod**
Mutli-Language support.
I'm not a LUA dev, you might add a TextField to get the language key from UI.

### CODE REVIEW

**Comment by omstrumpf** [comment]
Sorry it took me so long to get to this...

It turns out that handling multi-language is more complicated than it seems. Your solution is the best from the ones I've tried, but its not quite perfect. Here's why:

There are a bunch of cards that are available in multiple languages, but who's new printings are English-only. This is especially true around spoiler season. So we query for the newest version of the card (or whichever version the user specified), and find that it doesn't have a version in the language we want. However, if we had just checked the older printings, we may have found it.

So, why can't we just query Scryfall for all printings in the language we're looking for, and take the best one? There are two reasons. One is that Scryfall doesn't really to support this. The best we can do is query for all printings in all languages for a given card. This would be fine, but for the second reason. Which is that TTS straight up crashes if you get too big of a response back from a web request. Unfortunately Scryfall only supports pagination in groups of 175, which is too much for poor old TTS.

tl;dr we're currently missing older non-English printings, and TTS can't handle the bigger query we'd need to make to get around it.

**Comment by omstrumpf** [comment]
I'm gonna close this PR for now and continue discussion on #5, since its outdated now (which is completely my fault!). We can re-open in future if we make progress.
